src/components/BookAddForm.tsx

```diff
@@ -0,0 +1,30 @@
+import { memo } from 'react';
+import QuantityButtons, { QuantityButtonsProps } from './QuantityButtons';
+
+interface BookAddFormProps extends QuantityButtonsProps {
+    handleSubmit: (e: React.SyntheticEvent<HTMLFormElement>) => void,
+}
+
+const BookAddForm = ({
+	handleSubmit,
+	quantity,
+	handleQuantityChange,
+	handleQuantityInputChange,
+} : BookAddFormProps) => (
+	<form
+		className="book-add-form"
+		onSubmit={handleSubmit}
+	>
+		<div className="book-add-form__block">
+			<span>Quantity:</span>
+			<QuantityButtons
+				handleQuantityInputChange={handleQuantityInputChange}
```

Comment:
```suggestion
				onQuantityInputChange={handleQuantityInputChange}
```
ну и везде где найдёшь токое